chromium/browser_type.go

```diff
@@ -68,7 +68,12 @@ func (b *BrowserType) init(
 		return nil, nil, nil, fmt.Errorf("error setting up logger: %w", err)
 	}
 
-	launchOpts := common.NewLaunchOptions(k6ext.OnCloud(), isRemoteBrowser)
+	cloudMode := common.CloudModeOff
```

Comment:
Why do we need to differentiate the two? Aren't then the same now?

Reply:
Same comment as in https://github.com/grafana/xk6-browser/pull/800#discussion_r1121560746:

> But you say this based on the preliminary proposed solution for running browsers in the cloud right? Should we assume already that the browser is gonna be remote and therefore apply the changes?

Reply:
Resolving this for clarity as it's the same discussion as the one held in https://github.com/grafana/xk6-browser/pull/800#discussion_r1121487396.